partcad-cli/src/partcad_cli/click/commands/list/assemblies.py

```
@@ -0,0 +1,72 @@
+from partcad import logging
+from partcad.logging import Process
+
+
+@click.command(help="List available assemblies")
+@click.option("-r", "--recursive", is_flag=True, help="Recursively process all imported packages")
+@click.option(
+    "-u", "--used_by", type=str, required=False, help="Only process objects used by the given assembly or scene."
+)
+@click.argument("package", type=str, required=False)  # help="Package to retrieve the object from"
+@click.pass_obj
+def cli(ctx, recursive, used_by, package):
```

Ruff (0.8.2)

`cli` is too complex (13 > 10)

(C901)